patches/aesthetics.py

```diff
@@ -161,18 +161,95 @@ def reduceMessageLengths(rom, rnd):
     rom.texts[0x01] = formatText(rnd.choice([
         "Let's a go!",
         "Remember, sword goes on A!",
+        "Remember, sword goes on B!",
+        "It's pronounced Hydrocity Zone.",
         "Avoid the heart piece of shame!",
         "Marin? No, this is Zelda. Welcome to Hyrule",
         "Why are you in my bed?",
         "This is not a Mario game!",
         "MuffinJets was here...",
-        "Remember, there are no bugs in LADX",
+        "Wait, I thought Daid was French!",
+        "Is it spicefather or spaceotter?",
+        "kbranch finally took a break!",
+        "Pink Batman knows that trick, ask them!",
+        "Baby seed ahead.",
+        "Abandon all hope ye who enter here...",
+        "Link... Open your eyes...\nWait, you're #####?",
+        "Remember, there are no bugs in LADX.",
         "#####, #####, you got to wake up!\nDinner is ready.",
-        "Go find the stepladder",
+        "Go find the stepladder.",
         "Pizza power!",
-        "Eastmost penninsula is the secret",
-        "There is no cow level",
-        "You cannot lift rocks with your bear hands",
+        "Eastmost peninsula is the secret.",
+        "There is no cow level.",
+        "You cannot lift rocks with your bear hands.",
+        "Don't worry, the doghouse was patched.",
+        "The carpet whale isn't real, it can't hurt you.",
+        "Isn't this a demake of Phantom Hourglass?",
+        "GGs to TGH for WR.",
+        "Go try the LAS rando!",
+        "Go try the Oracles rando!",
+        "Go try Archipelago!",
+        "Go try touching grass!",
+        "Please leave my house.",
+        "Trust me, this will be a 2 hour seed, max.",
+        "This is still better than doing Dampe dungeons.",
+        "They say that Marin can be found here."
+        "Stalfos are such boneheads.",
+        "90% bug-free!",
+        "404 Marin.personality not found.",
+        "Idk man, works on my machine.",
+        "Hey guys, did you know that Vaporeon",
+        "Trans rights!",
+        "Support gay rights!\nAnd their lefts!",
+        "Snake? Snake?! SNAAAAKE!!!",
+        "Oh, you chose THESE settings?",
+        "As seen on TV!",
+        "May contain nuts.",
+        "Limited edition!",
+        "May contain RNG.",
+        "Reticulating splines!", 
+        "Keyboard compatible!",
+        "Teetsuuuuoooo!",
+        "Kaaneeeedaaaa!",
+        "Learn about allyship!",
+        "[This Marin text left intentionally blank.]",
+        "'Autological' is!",
+        "Technoblade never dies!",
+        "Thank you, CrystalSaver!",
+        "Wait, LADX has a rando?",
+        "https://tinyurl.com/ladxr",

```

Comment:
The official URL is shorter then the tiny url: `http://ladxr.daid.eu/`

Reply:
I know, the two links are different though 😉

Reply:
DAMN YOU. I'll allow it.